### ISSUE TITLE
Publish to maven central

### ISSUE THREAD

**Issue by cjnolet** [scalation/scalation#8]
Would be extremely useful if these artifacts were available on maven central.

**Comment by cjnolet** [comment]
Would you guys be opposed to me taking this ticket on? I've been on several projects now where we've published artifacts to Sonatype OSS. It's free and it gets stuff into Maven Central.

I'm interested in using some of the models provided in this project on a project that some Cloudera guys are working on to expose time series data in Apache Spark. It would be useful if the library was accessible through maven central so that end-users don't need to build it manually.

**Comment by mepcotterell** [comment]
I don't think it would be a good idea to do this until we're done with migration. However, I do see your point. In the scalation/analytics project, we're checking builds via Travis CI. This required us to make sure that ScalaTion is available as a dependency. We accomplished this by creating a script that downloads the latest version of the library, compiles it, and publishes it to a local Ivy repository.

Here are the related files:
 * https://github.com/scalation/analytics/blob/mepcotterell/.travis.yml
 * https://github.com/scalation/analytics/blob/mepcotterell/script/deps.sh
 * https://github.com/scalation/analytics/blob/mepcotterell/project/build.sbt

**Comment by antonkulaga** [comment]
Any progress on this? I have not worked with Maven Central but I can setup bintray publishing easily
